package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
According to https://docs.npmjs.com/files/package.json it will add 'The file in the "main" field' but it doesn't mention `module`. Plus there's other stuff in `dist/` like the minified version of the built file. So I'm pretty sure we still need `dist/` here.